src/base.css

```diff
@@ -1344,6 +1346,21 @@ Possible extra rowspan handling
 		font-weight: bold;
 	}
 
+/** Character examples (from the I18N template) *******************************/
+.codepoint bdi {
+    line-height: 1em;
+    font-size: 120%;
+    margin-inline: 0.25em;
+}
+
+.uname {
+    font-family: Arial, monospace;
```

Comment:
Setting the font family to Arial, monospace seems super weird. What's the justification?

Reply:
We like how Arial looks and know that it is widely available. It is different from the body font of a spec. When Arial is not available, falling back to a sans serif font is not necessarily going to produce the contrast desired. We dithered about just making the font _be_ monospace.

Reply:
Alternatively you could just let the font family inherit? It's defined to be `sans-serif`, which could be anything sans-serif -- including Arial.

You already have color and capitalization shifts here, I don't think a font family shift is also necessary (particularly given this text is reasonably understandable even if there was no formatting change at all!).

Reply:
Agreed. See below for further changes.